### DISCUSSION TITLE
Recent Outage

### DISCUSSION THREAD

**Discussion by Vortigern-The-Grey** [community/community#135828] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Anyone know anything about the very recent outage?

**Comment by BestLeaks** [comment, marked as answer]
If you checkout their status page incident: https://www.githubstatus.com/incidents/kz4khcgdsfdv

They state "We suspect the impact is due to a database infrastructure related change that we are working on rolling back."

and then they did the roll back and it was up again.

**Reply by Vortigern-The-Grey** [reply]
Same as last time? LMAO

**Reply by dshetty312** [reply]
yes it is.
